<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopterControlWidget</class> 
 <widget class="QWidget" name="CopterControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>frameSize</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="frameSizeSpinBox">
       <property name="toolTip">
         <string>frameSize</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>inverted</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="invertedCheckBox">
       <property name="toolTip">
         <string>inverted</string>
       </property>
       <property name="checked">
           <bool>0</bool>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>P Pitch</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="pPitchSpinBox">
       <property name="toolTip">
         <string>P Pitch</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>I Pitch</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="iPitchSpinBox">
       <property name="toolTip">
         <string>I Pitch</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>D Pitch</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QDoubleSpinBox" name="dPitchSpinBox">
       <property name="toolTip">
         <string>D Pitch</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>P Roll</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDoubleSpinBox" name="pRollSpinBox">
       <property name="toolTip">
         <string>P Roll</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
       <property name="text">
         <string>I Roll</string>
       </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QDoubleSpinBox" name="iRollSpinBox">
       <property name="toolTip">
         <string>I Roll</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
       <property name="text">
         <string>D Roll</string>
       </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="dRollSpinBox">
       <property name="toolTip">
         <string>D Roll</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
       <property name="text">
         <string>P Yaw</string>
       </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QDoubleSpinBox" name="pYawSpinBox">
       <property name="toolTip">
         <string>P Yaw</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
       <property name="text">
         <string>I Yaw</string>
       </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDoubleSpinBox" name="iYawSpinBox">
       <property name="toolTip">
         <string>I Yaw</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_10">
       <property name="text">
         <string>D Yaw</string>
       </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QDoubleSpinBox" name="dYawSpinBox">
       <property name="toolTip">
         <string>D Yaw</string>
       </property>
       <property name="prefix">
        <string></string>
       </property>
       <property name="suffix">
        <string></string>
       </property>
     </widget>
    </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>frameSizeSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>invertedCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>pPitchSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>iPitchSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>dPitchSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>pRollSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>iRollSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>dRollSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>pYawSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>iYawSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>dYawSpinBox</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CopterControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
